### PR TITLE
[feature] srote-ts

### PR DESCRIPTION
**Pull request by ArtemPapusha**

### CODE REVIEW

File: src/components/CardProduct/Component.ts [hunk 1]
```diff
@@ -104,17 +100,17 @@ class CardProduct implements CardProductInterface {
...
-      space[`px-3`],
-      space[`gap-10`],
-      space[`my-3`],
-      space[`mx-3`],
+      `d-flex`,
+      `just-content-flex-start`,
+      `align-items-center`,
+      `flex-direction-column`,
+      `flex-wrap-wrap`,
+      `wd-20`,
```
Comment (fatush55) [inline src/components/CardProduct/Component.ts:108]: use "" or '' string

File: src/components/CardProduct/Component.ts [hunk 2]
```diff
@@ -124,11 +120,11 @@ class CardProduct implements CardProductInterface {
...
     $footerCardProduct.className = [
       style['card-product__footer'],
-      flex[`d-flex`],
-      flex[`just-content-space-between`],
-      flex[`align-items-center`],
-      flex[`flex-direction-row`],
-      space[`gap-10`],
+      `d-flex`,
+      `just-content-space-between`,
+      `align-items-center`,
```
Comment (fatush55) [inline src/components/CardProduct/Component.ts:125]: use "" or '' string

File: src/components/Modal/Component.ts [hunk 3]
```diff
@@ -0,0 +1,156 @@
+import Button from "@elements/Button";
+import { $divApp } from "@constants/div.app";
+
+import {
+  type ModalImplements,
+  type ModalConstructor
+} from "./type";
+
+import style from './style.module.scss';
+import flex from '../../assets/style/utils/flex.module.scss';
```
Comment (fatush55) [inline src/components/Modal/Component.ts:10]: Why are you importing this here, we've  dicided to import the utility into page

File: src/components/Modal/style.module.scss [hunk 4]
```diff
@@ -0,0 +1,37 @@
+@import '@style/utils/colors.module.scss';
```
Comment (fatush55) [inline src/components/Modal/style.module.scss:1]: whu module.scss ?
we have diceded to create utils with name .scss

File: src/components/Pagination/Component.ts [hunk 5]
```diff
@@ -238,13 +218,13 @@ class Pagination implements PaginationInterface{
...
-    
+
     $paginationContainer.className = [
       style['pagination_container'],
-      flex[`d-flex`],
-      flex[`just-content-center`],
-      flex[`align-items-center`],
-      space[`gap-1`],
+      `d-flex`,
+      `just-content-center`,
```
Comment (fatush55) [inline src/components/Pagination/Component.ts:225]: Need to use "' or '' strin

File: src/elements/Icon/type.ts [hunk 6]
```diff
@@ -1,12 +1,13 @@
-import type { Color, IconName } from '@type/app.ts';
+import type { Color } from '@type/app.ts';
```
Comment (fatush55) [inline src/elements/Icon/type.ts:1]: remove ".ts", use just "@type/app"

File: src/elements/Skeleton/Component.ts [hunk 7]
```diff
@@ -11,149 +8,131 @@ class Skeleton implements SkeletonInterface {
...
+    $titleSkeleton.className = [
+      style.title_skeleton,
+      'wd-100p',
+    ].join(' ');
   
+    for (let i = 0; i < 3; i++) {
+      const $titleSkeletonElement = document.createElement('div');
+
       $titleSkeletonElement.className = [
         style.title_skeleton_element,
```
Comment (fatush55) [inline src/elements/Skeleton/Component.ts:78]: use cameCase

File: src/elements/Typography/Component.ts [hunk 8]
```diff
@@ -55,12 +53,10 @@ class Typography implements TypographyInterface {
     $typography.className = [
       style.typography,
       style[`typography-${this.type}`],
-      colors[`text-${this.color}`],
-      fonts[`fw-${this.textWeight}`],
+      `text-${this.color}`,
```
Comment (fatush55) [inline src/elements/Typography/Component.ts:56]: use "" or '' string

File: src/elements/Skeleton/Component.ts (hunk 7 above)
Comment (ArtemPapusha) [reply src/elements/Skeleton/Component.ts:78]: for class name?

File: src/elements/Typography/Component.ts (hunk 8 above)
Comment (ArtemPapusha) [reply src/elements/Typography/Component.ts:56]: `text-${this.color}` this need only ``, with using ${}

**Pull request merged**